CHANGELOG.md

```diff
@@ -3,14 +3,15 @@
 ## master / unreleased
 
 * [CHANGE] Changed default for `-ingester.min-ready-duration` from 1 minute to 15 seconds. #4539
-* [ENHANCEMENT] Ruler: Add `ruler.disable-rule-group-label` to disable the `rule_group` label on exported metrics. #4571
+* [CHANGE] query-frontend: Do not print anything in the logs of `query-frontend` if a in-progress query has been canceled (context canceled). #4562
```

Comment:
Unrelated change - please move to a different PR.

Reply:
Posted #4584